src/index.html

```diff
@@ -5,8 +5,328 @@
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+          Отзывов: 5
+        </div>
+        <div class="card__price">
+            Цена:
+          <span class="card__sum">
+            69 999 грн
+          </span>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          купить
+        </a>
+      </div>
+
+      <div class="products__card card">
+        <div class="card__img"></div>
+        <h2 class="card__name">
+          Моноблок APPLE A1419 iMac
+          27" Retina 5K (MNED2UA/A)
+        </h2>
+        <span class="card__code">
+          Код товара: 195434
+        </span>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+          Отзывов: 5
+        </div>
+        <div class="card__price">
+            Цена:
+          <span class="card__sum">
+            69 999 грн
+          </span>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          купить
+        </a>
+      </div>
+
+      <div class="products__card card">
+        <div class="card__img"></div>
+        <h2 class="card__name">
+          Моноблок APPLE A1419 iMac
+          27" Retina 5K (MNED2UA/A)
+        </h2>
+        <span class="card__code">
+          Код товара: 195434
+        </span>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+          Отзывов: 5
+        </div>
+        <div class="card__price">
+            Цена:
+          <span class="card__sum">
+            69 999 грн
+          </span>
+        </div>
+        <a
+          href="#"
+          class="card__buy"
+        >
+          купить
+        </a>
+      </div>
+
+      <div class="products__card card">
+        <div class="card__img"></div>
+        <h2 class="card__name">
+          Моноблок APPLE A1419 iMac
+          27" Retina 5K (MNED2UA/A)
+        </h2>
+        <span class="card__code">
+          Код товара: 195434
+        </span>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+          Отзывов: 5
+        </div>
```

Comment:
Тут можна незалежний блок окремо відділити
```suggestion
        <div class="card__rating">
        
          <div class="stars stars--4">
            <div class="stars__item"></div>
            <div class="stars__item"></div>
            <div class="stars__item"></div>
            <div class="stars__item"></div>
            <div class="stars__item"></div>
          </div>
          
          Отзывов: 5
        </div>
```
Хоча не впевнений чи з 2-х боків треба це робити)